<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MnInfoDialog</class>
 <widget class="QWidget" name="MnInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>574</width>
    <height>530</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="labelTitle">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">padding-left:24px;</string>
          </property>
          <property name="text">
           <string>Master Node Information</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>7</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnEsc">
          <property name="minimumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">border:none;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="layoutScroll" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="styleSheet">
            <string notr="true">#scrollArea {
background:transparent;
}</string>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustIgnored</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>570</width>
              <height>596</height>
             </rect>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">#scrollAreaWidgetContents {
background:transparent;
}</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QWidget" name="content" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <property name="leftMargin">
                 <number>16</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>16</number>
                </property>
                <item>
                 <widget class="QWidget" name="contentID" native="true">
                  <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="0,1,0">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="labelId">
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Public Key:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="textId">
                     <property name="text">
                      <string>492526e7fa3c810b35016...40a5df85ee227ab00b1156994</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pushCopy">
                     <property name="minimumSize">
                      <size>
                       <width>34</width>
                       <height>34</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>34</width>
                       <height>34</height>
                      </size>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>24</width>
                       <height>24</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelDivider9">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>1</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color:#bababa;</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="contentOutputs" native="true">
                  <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="0,3">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>12</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="labelSend">
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Address: </string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="textAddress">
                     <property name="text">
                      <string>127.0.0.2:43223</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelDivider8">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>1</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color:#bababa;</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="contentAmount" native="true">
                  <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1,0">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="labelAmount">
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Collaterall tx id:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="textAmount">
                     <property name="text">
                      <string>492526e7fa3c810b35016...40a5df85ee227ab00b1156994</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pushCopyId">
                     <property name="minimumSize">
                      <size>
                       <width>34</width>
                       <height>34</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>34</width>
                       <height>34</height>
                      </size>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelDivider6">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>1</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color:#bababa;</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="contentInputs" native="true">
                  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,2">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>12</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="labelInputs">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Output index:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="textInputs">
                     <property name="text">
                      <string>1 Inputs</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelDivider1">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>1</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color:#bababa;</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="contentFee" native="true">
                  <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,1">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="labelFee">
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Status:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="textStatus">
                     <property name="text">
                      <string>MISSING</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelDivider4">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>1</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color:#bababa;</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="contentExport" native="true">
                  <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="3,0">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="textExport">
                     <property name="styleSheet">
                      <string notr="true">padding-top:1px;</string>
                     </property>
                     <property name="lineWidth">
                      <number>0</number>
                     </property>
                     <property name="text">
                      <string>Export data to run the Master Node on a remote server</string>
                     </property>
                     <property name="wordWrap">
                      <bool>true</bool>
                     </property>
                     <property name="indent">
                      <number>0</number>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pushExport">
                     <property name="minimumSize">
                      <size>
                       <width>34</width>
                       <height>34</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>34</width>
                       <height>34</height>
                      </size>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelDivider7">
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>1</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">background-color:#bababa;</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
